packages/appstore/src/components/cfds-listing/cfds-listing.scss

```diff
@@ -675,6 +721,86 @@
     }
 }
 
+.dynamic-leverage-modal {
+    display: flex;
+    flex-direction: column;
+    gap: 2.4rem;
+    margin: 1.5rem 12.8rem 10rem;
+
+    @include mobile {
+        margin: 1.5rem 2.4rem 2.4rem;
+    }
+
+    position: absolute;
```

Comment:
@arshad-rao-deriv let's put all styles of `.dynamic-leverage-modal` class together, so that `@include` directive for mobile is not splitting them :) And please also check for extra line breaks in this file. Please press Option+Shift+F to ensure this file is prettified.🙏🏻

Reply:
Done.